Extension/src/LanguageServer/settings.ts

```diff
@@ -414,337 +517,157 @@ export class CppSettings extends Settings {
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.space.aroundBinaryOperator")!;
+        return this.getAsString("vcFormat.space.aroundBinaryOperator")!;
     }
-
     public get vcFormatSpaceAroundAssignmentOperator(): string {
         // These strings have default values in package.json, so should never be undefined.
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.space.aroundAssignmentOperator")!;
+        return this.getAsString("vcFormat.space.aroundAssignmentOperator")!;
     }
-
     public get vcFormatSpacePointerReferenceAlignment(): string {
         // These strings have default values in package.json, so should never be undefined.
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.space.pointerReferenceAlignment")!;
+        return this.getAsString("vcFormat.space.pointerReferenceAlignment")!;
     }
-
     public get vcFormatSpaceAroundTernaryOperator(): string {
         // These strings have default values in package.json, so should never be undefined.
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.space.aroundTernaryOperator")!;
+        return this.getAsString("vcFormat.space.aroundTernaryOperator")!;
     }
-
     public get vcFormatWrapPreserveBlocks(): string {
         // These strings have default values in package.json, so should never be undefined.
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return super.Section.get<string>("vcFormat.wrap.preserveBlocks")!;
-    }
-
-    public get dimInactiveRegions(): boolean {
-        return super.Section.get<boolean>("dimInactiveRegions") === true
-            && this.intelliSenseEngine === "default"
-            && vscode.workspace.getConfiguration("workbench").get<string>("colorTheme") !== "Default High Contrast";
-    }
-
-    public get sshTargetsView(): string {
-        return super.Section.get<string>("sshTargetsView") ?? 'default';
+        return this.getAsString("vcFormat.wrap.preserveBlocks")!;
     }
+    public get dimInactiveRegions(): boolean { return this.getAsBoolean("dimInactiveRegions") && this.intelliSenseEngine === "default" && vscode.workspace.getConfiguration("workbench").get<string>("colorTheme") !== "Default High Contrast"; }
+    public get sshTargetsView(): string { return this.getAsString("sshTargetsView") ?? 'default'; }
 
     public toggleSetting(name: string, value1: string, value2: string): void {
-        const value: string | undefined = super.Section.get<string>(name);
+        const value: string | undefined = this.getAsString(name);
         void super.Section.update(name, value?.toLowerCase() === value1.toLowerCase() ? value2 : value1, getTarget());
```

Comment:
`value` should already be lower-case, but I'm not sure if we have a way to assert that the setting was an enum

Reply:
Recalling this comment. We can't make getAsString do the lower-casing as explained in another comment.